utils/layoutHasPaint.test.js

```
@@ -0,0 +1,19 @@
+const {layoutHasPaint} = require('./layoutHasPaint');
+const test = require('node:test');
+const assert = require('node:assert/strict');
+
+test('empty home header is not a painted page', () => {
+  assert.equal(
+    layoutHasPaint({root: {header: {rows: []}}}),
+    false,
+  );
```

ESLint

[error] Replace `⏎····layoutHasPaint({root:·{header:·{rows:·[]}}}),⏎····false,⏎··` with `layoutHasPaint({root:·{header:·{rows:·[]}}}),·false`

(prettier/prettier)